client/devdocs/design/component-playground.jsx

```diff
@@ -42,10 +43,14 @@ class ComponentPlayground extends Component {
 			'design__component-playground-code': true,
 			'show-code': toggleCode ? this.state.showCode : true,
 		} );
+		let scope = playgroundScopeForComponents;
+		if ( 'gutenberg-components' === this.props.section ) {
+			scope = playgroundScopeForGutenbergComponents;
+		}
```

Comment:
How about a ternary to avoid the let?
```
const scope = 'gutenberg-components' === this.props.section ? playgroundScopeForGutenbergComponents : playgroundScopeForComponents;
```